app/views/shared/_project_summary.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <span class="govuk-caption-l">URN <%= @project.urn %></span>
+    <h1 class="govuk-heading-l"><%= @project.establishment.name %></h1>
+
+    <%= govuk_summary_list(actions: false) do |summary_list|
+        summary_list.row do |row|
+          row.key { t('project.summary.proposed_opening_date.title') }
+          row.value { Date.parse(@project.conversion_project.proposed_opening_date).strftime("%-d %B %Y") }
```

Comment:
Just having a quick glance through this.

Remembered from SE that we can define our date formats in a nice way like this ⌚  https://github.com/DFE-Digital/schools-experience/blob/a648a489751f9c5eff9519dbb7eaa4e51b0bf766/config/initializers/date_format.rb

Reply:
I _knew_ I'd seen this done before somewhere!